filer/admin/folderadmin.py

```diff
@@ -1076,19 +1076,19 @@ def _copy_folder(self, folder, destination, suffix, overwrite):
 
         old_folder = Folder.objects.get(pk=folder.pk)
 
-        # Due to how inheritance works, we have to set both pk and id to None
-        folder.pk = None
-        folder.id = None
-        folder.name = foldername
-        folder.insert_at(destination, 'last-child', True)  # We save folder here
+        new_folder = Folder.objects.create(
```

Comment:
I think doing 'get_or_create' is getter to prevent exceptions. What do you think?

Reply:
We can also just keep using folder as the variable name, it doesn't really make any difference other than the semantics.

Reply:
@fsbraun Do you agree?

Reply:
@vinitkumar @fsbraun Changed with `get_or_create`, it does look better.